packages/playground/src/components/mosha_toast/offline_nodes_toast.vue

```diff
@@ -4,10 +4,11 @@
     <button class="text-decoration-underline" :onclick="navigateToContractsList">
       {{ deploymentLen }} {{ deploymentLen > 1 ? "contracts" : "contract" }}
     </button>
-    on offline or standby {{ deploymentLen > 1 ? "nodes" : "node" }}
+    on offline {{ deploymentLen > 1 ? "nodes" : "node" }}.
     <v-spacer />
     <span v-if="withPubIpLen >= 1">
-      {{ withPubIpLen }} {{ deploymentLen > 1 ? "of them" : "" }} with public {{ withPubIpLen > 1 ? "IPs" : "IP" }}
+      and {{ withPubIpLen }} {{ deploymentLen > 1 ? "of them" : "" }} with public {{ withPubIpLen > 1 ? "IPs" : "IP" }}.
```

Comment:
```suggestion
       {{ withPubIpLen }} {{ deploymentLen > 1 ? "of them" : "" }} with public {{ withPubIpLen > 1 ? "IPs" : "IP" }}.
```

Reply:
should be resolved in https://github.com/threefoldtech/tfgrid-sdk-ts/pull/1644/commits/65aa20498ace8655d59bdf3f827ee7fde9f07c79